### PR TITLE
Update to latest FFSVM 0.12, making visqol compile on stable.

### PR DESCRIPTION
**Pull request by ralfbiedert**
Thanks for using FFSVM, author here.

Up until just now FFSVM required Rust nightly. With the new 0.12 it compiles on stable. This PR updates the dependency inside visqol. With this you should compile on stable Rust 1.83+ (I've tested this on my machine). I can also report it works on Windows, at least according to `cargo test`.

### CODE REVIEW

File: visqol/Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ example = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-visqol-rs = "0.1.1"
+visqol-rs = { path = "../visqol-rs" }
```
Comment (dstrub18) [inline visqol/Cargo.toml:22]: I would prefer to keep the binary decoupled from the library. Could you revert this change?

**Review by dstrub18 — COMMENTED** [review]
Hey there!
Wow, thank you for so much effort into making ffsvm run on stable. Really great work.
Thank you for bumping the ffsvm version and fixing the API calls.
I'd only ask you to keep the dependency in `visqol/Cargo.toml` as it is right now, so that I can bump that separately.

Thank you either way!

File: visqol/Cargo.toml (hunk 1 above)
Comment (ralfbiedert) [reply visqol/Cargo.toml]: Sure!

**Comment by ralfbiedert** [comment]
> I'd only ask you to keep the dependency in visqol/Cargo.toml as it is right now, so that I can bump that separately.

Done. Please let me know if there is anything else.

**Review by dstrub18 — APPROVED** [review]
Great, thank you! Happy holidays in case you're celebrating
